tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -83,7 +92,10 @@ func (s *stresser) Stress() error {
 					Value: []byte(randStr(s.KeySize)),
 				})
 				putcancel()
-				if grpc.ErrorDesc(err) == context.Canceled.Error() {
+				if grpc.ErrorDesc(err) == context.Canceled.Error() ||
```

Comment:
Ok will make it into a function. (just tested and checked that we need to check all those)